browser/brave_ads/search_result_ad/search_result_ad_handler_aliases.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_BRAVE_ADS_SEARCH_RESULT_AD_SEARCH_RESULT_AD_HANDLER_ALIASES_H_
+#define BRAVE_BROWSER_BRAVE_ADS_SEARCH_RESULT_AD_SEARCH_RESULT_AD_HANDLER_ALIASES_H_
+
+#include <vector>
+
+#include "base/callback.h"
+#include "brave/vendor/bat-native-ads/include/bat/ads/public/interfaces/ads.mojom.h"
+
+namespace content {
+class RenderFrameHost;
+}
+
+namespace brave_ads {
+
+using SearchResultAdsList = std::vector<ads::mojom::SearchResultAdPtr>;
```

Comment:
this seems like an odd place for these. This `_aliases.h` convention is only used in bat ads right now and it's best to stick with established upstream conventions to make it easier for people to find and understand things. Not to mention that putting them here makes it impossible to reference them from the component where they will eventually be needed

Reply:
Agreed, I will move it to `search_result_ad_handler.h` back. 
cc @tmancey

Reply:
I have added `search_result_ad_aliases.h` back.
Because it now contains `SearchResultAdsList` type which is used in two headers: `search_result_ad_handler.h` and `parse_web_page_entities.h`

Reply:
even if we keep search_result_ad_util which I'm not sure why we need to https://github.com/brave/brave-core/pull/12816#discussion_r852326984 then it can go in one or the other of them. There's no reason to add a 3rd header for this, but seems like it should all just go in search_result_ad_handler

Reply:
this `_aliases` pattern should not be used in general

Reply:
removed `search_result_ad_aliases.h` file